### DISCUSSION TITLE
Cannot convert error  message

### DISCUSSION THREAD

**Discussion by colc1** [apprenticeharper/DeDRM_tools#1972] [Q&A]
After 7 years and 688 downloaded books I have finally struck a snag with Calibre.
I received an error message" cannot convert Crusaders Cross, locked by DRM"
I would appreciate any comments.
Apologies if I have posted in the wrong area.

**Comment by ElleKayEm** [comment]
Will need more info to help.  Let's take a look at a log:  https://github.com/apprenticeharper/DeDRM_tools/blob/master/FAQs.md#i-cannot-solve-my-problem-with-the-dedrm-plugin-and-now-i-need-to-post-a-log-how-do-i-do-that

**Comment by colc1** [comment]
I hope I have completed this procedure correctly.

Col

 

 

calibre Debug log

calibre 5.33.2  embedded-python: True is64bit: False

Windows-10-10.0.19041 Windows ('32bit', 'WindowsPE')

32bit process running on 64bit windows

('Windows', '10', '10.0.19041')

Python 3.8.5

Windows: ('10', '10.0.19041', '', 'Multiprocessor Free')

Interface language: None

Successfully initialized third party plugins: 

calibre 5.33.2  embedded-python: True is64bit: False

Windows-10-10.0.19041 Windows ('32bit', 'WindowsPE')

32bit process running on 64bit windows

('Windows', '10', '10.0.19041')

Python 3.8.5

Windows: ('10', '10.0.19041', '', 'Multiprocessor Free')

Interface language: None

Successfully initialized third party plugins: 

Turning on automatic hidpi scaling

devicePixelRatio: 1.0

logicalDpi: 96.0 x 96.0

physicalDpi: 93.6046065259117 x 93.62457337883959

Using calibre Qt style: True

[0.00] Starting up...

[0.00] Showing splash screen...

[0.53] splash screen shown

[0.53] Initializing db...

[0.58] db initialized

[0.58] Constructing main UI...

[1.61] main UI initialized...

[1.61] Hiding splash screen

Starting QuickView

[75.89] splash screen hidden

[75.89] Started up in 75.89 seconds with 667 books

stdout+stderr from file dialog helper: [b'', b'']

piped data from file dialog helper: [b'\x1f\xae\x9d\x03j\xf4x\xaa\xfa\xb8@\xb0\xcc\t\xa0T]9\xb5\x0c\x9b\xa80\x9f\xf1M\xbe\xe1\x9f\xaae\x1e', b'C:\\Users\\Col\\Documents\\My Digital Editions\\The Crusaders Cross.epub']

Added The Crusader's Cross to db in: 1.4

Failed to initialize plugin: 'C:\\Users\\Col\\AppData\\Roaming\\calibre\\plugins\\DeDRM.zip'

Added 1 books in 1.9 seconds

Worker Launch took: 0.11 seconds

Job: 0 Convert book 1 of 1 (The Crusader's Cross) finished

Convert book 1 of 1 (The Crusader's Cross)

          Conversion options changed from defaults:

            verbose: 2

            read_metadata_from_opf: 'C:\\Users\\Col\\AppData\\Local\\Temp\\calibre_3uuqjv0n\\jqnveb1e.opf'

          Resolved conversion options

          calibre version: 5.33.2

          {'asciiize': False,

          'author_sort': None,

          'authors': None,

          'base_font_size': 0.0,

          'book_producer': None,

          'change_justification': 'original',

          'chapter': "//*[((name()='h1' or name()='h2') and re:test(., "

                      "'\\s*((chapter|book|section|part)\\s+)|((prolog|prologue|epilogue)(\\s+|$))', "

                      "'i')) or @class = 'chapter']",

          'chapter_mark': 'pagebreak',

          'comments': None,

          'cover': None,

          'debug_pipeline': None,

          'dehyphenate': True,

          'delete_blank_paragraphs': True,

          'disable_font_rescaling': False,

          'dont_compress': False,

          'duplicate_links_in_toc': False,

          'embed_all_fonts': False,

          'embed_font_family': None,

          'enable_heuristics': False,

          'expand_css': False,

          'extra_css': None,

          'extract_to': None,

          'filter_css': '',

          'fix_indents': True,

          'font_size_mapping': None,

          'format_scene_breaks': True,

          'html_unwrap_factor': 0.4,

          'input_encoding': None,

          'input_profile': <calibre.customize.profiles.InputProfile object at 0x06F1E1A8>,

          'insert_blank_line': False,

          'insert_blank_line_size': 0.5,

          'insert_metadata': False,

          'isbn': None,

          'italicize_common_cases': True,

          'keep_ligatures': False,

          'language': None,

          'level1_toc': None,

          'level2_toc': None,

          'level3_toc': None,

          'line_height': 0.0,

          'linearize_tables': False,

          'margin_bottom': 5.0,

          'margin_left': 5.0,

          'margin_right': 5.0,

          'margin_top': 5.0,

          'markup_chapter_headings': True,

          'max_toc_links': 50,

          'minimum_line_height': 120.0,

          'mobi_file_type': 'old',

          'mobi_ignore_margins': False,

          'mobi_keep_original_images': False,

          'mobi_toc_at_start': False,

          'no_chapters_in_toc': False,

          'no_inline_navbars': False,

          'no_inline_toc': False,

          'output_profile': <calibre.customize.profiles.OutputProfile object at 0x06F1E310>,

          'page_breaks_before': '/',

          'personal_doc': '[PDOC]',

          'prefer_author_sort': False,

          'prefer_metadata_cover': False,

          'pretty_print': False,

          'pubdate': None,

          'publisher': None,

          'rating': None,

          'read_metadata_from_opf': 'C:\\Users\\Col\\AppData\\Local\\Temp\\calibre_3uuqjv0n\\jqnveb1e.opf',

          'remove_fake_margins': True,

          'remove_first_image': False,

          'remove_paragraph_spacing': False,

          'remove_paragraph_spacing_indent_size': 1.5,

          'renumber_headings': True,

          'replace_scene_breaks': '',

          'search_replace': '[]',

          'series': None,

          'series_index': None,

          'share_not_sync': False,

          'smarten_punctuation': False,

          'sr1_replace': None,

          'sr1_search': None,

          'sr2_replace': None,

          'sr2_search': None,

          'sr3_replace': None,

          'sr3_search': None,

          'start_reading_at': None,

          'subset_embedded_fonts': False,

          'tags': None,

          'timestamp': None,

          'title': None,

          'title_sort': None,

          'toc_filter': None,

          'toc_threshold': 6,

          'toc_title': None,

          'transform_css_rules': '[]',

          'transform_html_rules': '[]',

          'unsmarten_punctuation': False,

          'unwrap_lines': True,

          'use_auto_toc': False,

          'verbose': 2}

          Failed to initialize plugin: 'C:\\Users\\Col\\AppData\\Roaming\\calibre\\plugins\\DeDRM.zip'

          InputFormatPlugin: EPUB Input running

          on C:\Users\Col\AppData\Local\Temp\calibre_3uuqjv0n\f2lguwhx.epub

          Traceback (most recent call last):

            File "runpy.py", line 194, in _run_module_as_main

            File "runpy.py", line 87, in _run_code

            File "site.py", line 82, in <module>

            File "site.py", line 77, in main

            File "site.py", line 49, in run_entry_point

            File "calibre\utils\ipc\worker.py", line 216, in main

            File "calibre\gui2\convert\gui_conversion.py", line 41, in gui_convert_override

            File "calibre\gui2\convert\gui_conversion.py", line 28, in gui_convert

            File "calibre\ebooks\conversion\plumber.py", line 1111, in run

            File "calibre\customize\conversion.py", line 244, in __call__

            File "calibre\ebooks\conversion\plugins\epub_input.py", line 287, in convert

          calibre.ebooks.DRMError: f2lguwhx.epub

 

stdout+stderr from file dialog helper: [b'', b'']

piped data from file dialog helper: [b'`\xad\xa0f\xf8\x9d\x8cA\x1f\xd3\x01\x16\xf8\xc8\x1e5&\x98N\xe7\x01\xb3\x19B\xa4\xd7\x1cr\x8f3\xfd\x96', b'C:\\Users\\Col\\Documents\\My Digital Editions\\The Crusaders Cross.epub']

Added The Crusader's Cross to db in: 1.2

Failed to initialize plugin: 'C:\\Users\\Col\\AppData\\Roaming\\calibre\\plugins\\DeDRM.zip'

Failed to initialize plugin: 'C:\\Users\\Col\\AppData\\Roaming\\calibre\\plugins\\DeDRM.zip'

Traceback (most recent call last):

  File "calibre\customize\ui.py", line 740, in initialize_plugins

  File "calibre\customize\ui.py", line 64, in load_plugin

  File "calibre\customize\zipplugin.py", line 293, in load

  File "importlib\__init__.py", line 127, in import_module

  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import

  File "<frozen importlib._bootstrap>", line 991, in _find_and_load

  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked

  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked

  File "calibre\customize\zipplugin.py", line 185, in exec_module

  File "calibre\customize\zipplugin.py", line 181, in get_code

  File "calibre_plugins.dedrm.__init__", line 149

    print u"{0} v{1}: Copying needed library files from plugin's zip".format(PLUGIN_NAME, PLUGIN_VERSION)

          ^

SyntaxError: invalid syntax

Added 1 books in 1.7 seconds

Worker Launch took: 0.11 seconds

Job: 1 Convert book 1 of 1 (The Crusader's Cross) finished

Convert book 1 of 1 (The Crusader's Cross)

          Conversion options changed from defaults:

            verbose: 2

            read_metadata_from_opf: 'C:\\Users\\Col\\AppData\\Local\\Temp\\calibre_3uuqjv0n\\4v1nzcp6.opf'

          Resolved conversion options

          calibre version: 5.33.2

          {'asciiize': False,

          'author_sort': None,

          'authors': None,

          'base_font_size': 0.0,

          'book_producer': None,

          'change_justification': 'original',

          'chapter': "//*[((name()='h1' or name()='h2') and re:test(., "

                      "'\\s*((chapter|book|section|part)\\s+)|((prolog|prologue|epilogue)(\\s+|$))', "

                     "'i')) or @class = 'chapter']",

          'chapter_mark': 'pagebreak',

          'comments': None,

          'cover': None,

          'debug_pipeline': None,

          'dehyphenate': True,

          'delete_blank_paragraphs': True,

          'disable_font_rescaling': False,

          'dont_compress': False,

          'duplicate_links_in_toc': False,

          'embed_all_fonts': False,

          'embed_font_family': None,

          'enable_heuristics': False,

          'expand_css': False,

          'extra_css': None,

          'extract_to': None,

          'filter_css': '',

          'fix_indents': True,

          'font_size_mapping': None,

          'format_scene_breaks': True,

          'html_unwrap_factor': 0.4,

          'input_encoding': None,

          'input_profile': <calibre.customize.profiles.InputProfile object at 0x0778E1A8>,

          'insert_blank_line': False,

          'insert_blank_line_size': 0.5,

          'insert_metadata': False,

          'isbn': None,

          'italicize_common_cases': True,

          'keep_ligatures': False,

          'language': None,

          'level1_toc': None,

          'level2_toc': None,

          'level3_toc': None,

          'line_height': 0.0,

          'linearize_tables': False,

          'margin_bottom': 5.0,

          'margin_left': 5.0,

          'margin_right': 5.0,

          'margin_top': 5.0,

          'markup_chapter_headings': True,

          'max_toc_links': 50,

          'minimum_line_height': 120.0,

          'mobi_file_type': 'old',

          'mobi_ignore_margins': False,

          'mobi_keep_original_images': False,

          'mobi_toc_at_start': False,

          'no_chapters_in_toc': False,

          'no_inline_navbars': False,

          'no_inline_toc': False,

          'output_profile': <calibre.customize.profiles.OutputProfile object at 0x0778E310>,

          'page_breaks_before': '/',

          'personal_doc': '[PDOC]',

          'prefer_author_sort': False,

          'prefer_metadata_cover': False,

          'pretty_print': False,

          'pubdate': None,

          'publisher': None,

          'rating': None,

          'read_metadata_from_opf': 'C:\\Users\\Col\\AppData\\Local\\Temp\\calibre_3uuqjv0n\\4v1nzcp6.opf',

          'remove_fake_margins': True,

          'remove_first_image': False,

          'remove_paragraph_spacing': False,

          'remove_paragraph_spacing_indent_size': 1.5,

          'renumber_headings': True,

          'replace_scene_breaks': '',

          'search_replace': '[]',

          'series': None,

          'series_index': None,

          'share_not_sync': False,

          'smarten_punctuation': False,

          'sr1_replace': None,

          'sr1_search': None,

          'sr2_replace': None,

          'sr2_search': None,

          'sr3_replace': None,

          'sr3_search': None,

          'start_reading_at': None,

          'subset_embedded_fonts': False,

          'tags': None,

          'timestamp': None,

          'title': None,

          'title_sort': None,

          'toc_filter': None,

          'toc_threshold': 6,

          'toc_title': None,

          'transform_css_rules': '[]',

          'transform_html_rules': '[]',

          'unsmarten_punctuation': False,

          'unwrap_lines': True,

          'use_auto_toc': False,

          'verbose': 2}

          Failed to initialize plugin: 'C:\\Users\\Col\\AppData\\Roaming\\calibre\\plugins\\DeDRM.zip'

          InputFormatPlugin: EPUB Input running

          on C:\Users\Col\AppData\Local\Temp\calibre_3uuqjv0n\kvn1vkl2.epub

          Traceback (most recent call last):

            File "runpy.py", line 194, in _run_module_as_main

            File "runpy.py", line 87, in _run_code

            File "site.py", line 82, in <module>

            File "site.py", line 77, in main

            File "site.py", line 49, in run_entry_point

            File "calibre\utils\ipc\worker.py", line 216, in main

            File "calibre\gui2\convert\gui_conversion.py", line 41, in gui_convert_override

            File "calibre\gui2\convert\gui_conversion.py", line 28, in gui_convert

            File "calibre\ebooks\conversion\plumber.py", line 1111, in run

            File "calibre\customize\conversion.py", line 244, in __call__

            File "calibre\ebooks\conversion\plugins\epub_input.py", line 287, in convert

          calibre.ebooks.DRMError: kvn1vkl2.epub

**Reply by ElleKayEm** [reply]
> Failed to initialize plugin: 'C:\\Users\\Col\\AppData\\Roaming\\calibre\\plugins\\DeDRM.zip

Try reinstalling DeDRM and then restarting calibre before trying to import a book.

**Comment by colc1** [comment]
Many thanks, that fixed the problem after downloading, FWIW the DeDRM file had disappeared from my PC.

**Reply by ElleKayEm** [reply]
Thanks for reporting back.
